contracts/starknet/space/space.cairo

```diff
@@ -0,0 +1,134 @@
+%lang starknet
+
+from starkware.starknet.common.syscalls import ( get_caller_address, get_block_number )
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.math import assert_lt, assert_le, assert_nn, assert_not_zero
+
+struct Vote:
```

Comment:
Implement these structs in a different file so that we can use them in other contracts

Reply:
https://github.com/snapshot-labs/sx-core/pull/47/commits/95e92d70a9eaa494b6b5c70eb2c6cbfaf56cf4fd :D